tensorzero-core/src/config_parser/path.rs

```diff
@@ -590,8 +654,11 @@ mod tests {
             DeTable::parse(r#"functions.my_function.system_schema = "relative/schema_path.json""#)
                 .unwrap();
 
-        let resolved =
-            resolve_toml_relative_paths(table.into_inner(), Path::new("my/base/path")).unwrap();
+        let resolved = resolve_toml_relative_paths(
```

Comment:
I think we might wanna add a test here for if there are multiple entries in the map?